engines/mohawk/myst_state.cpp

```diff
@@ -259,10 +259,14 @@ bool MystGameState::saveMetadata(int slot) {
 bool MystGameState::isAutoSaveAllowed() {
 	// Open autosave slot and see if it an autosave
 	// Autosaving will be enabled if it is an autosave or if there is no save in that slot
+	if (!loadState(kAutoSaveSlot)) { // Cannot load non-meta file, enable autosave
```

Comment:
But if loading succeeds, we will have overwritten the current state with the save in slot 0.